src/routing_utils/syslog_utils.sh

```diff
@@ -7,10 +7,10 @@ function tee_output_to_sys_log() {
   declare log_dir="$1"
   declare log_name="$2"
 
-  exec > >(tee -a >(logger -p user.info -t "vcap.${log_name}.stdout") | prepend_datetime >>"${log_dir}/${log_name}.log")
-  exec 2> >(tee -a >(logger -p user.error -t "vcap.${log_name}.stderr") | prepend_datetime >>"${log_dir}/${log_name}.err.log")
+  exec 1> >(tee -a >(log_prefix "O" >> "${log_dir}/${log_name}.log") | logger -p user.info -t "vcap.${log_name}.stdout")
```

Comment:
I would also remove stdout and stderr suffixes from logger `tag` as different `priority` should be enough however I'm not sure if it is ok for everybody.